### PR TITLE
Add new resource server using the JWT verifier

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

File: resource-server/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Okta Resource Server Example
+
+This sample application authenticates requests, using an OAuth 2.0 access tokens and [Okta JWT Verifier for Java][]
+
+The access tokens are obtained via the [Implicit Flow][].  One easy way to obtain an access token is to use the [OpenID Connect Debugger][]. Take a look at this [blog post][].
```
Comment (wayneearl-okta) [inline resource-server/README.md]: Is there supposed to be a link for the 'blog post'?

File: resource-server/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Prerequisites
+
+Before running this sample, you will need the following:
+
+* An Okta Developer Account, you can sign up for one at https://developer.okta.com/signup/.
+* An access token, obtained with the [OpenID Connect Debugger][] (instructions via this [blog post][])
```
Comment (wayneearl-okta) [inline resource-server/README.md]: And this one?

File: resource-server/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      "text": "Hello, world!"
+    }
+  ]
+}
+```
+
+[Implicit Flow]: https://developer.okta.com/authentication-guide/implementing-authentication/implicit
+[Okta JWT Verifier for Java]: https://github.com/okta/okta-jwt-verifier-java
+[OpenID Connect Debugger]: https://oidcdebugger.com/
+[blog post for instructions]: https://developer.okta.com/blog/2018/12/18/secure-spring-rest-api#generate-tokens-in-your-spring-rest-api
```
Comment (wayneearl-okta) [inline resource-server/README.md]: I see this reference, but it doesn't seem to work for the 'blog post' links above.

Comment (bdemers) [reply resource-server/README.md]: ahh, that one does look wrong, i'll fix it

**Pull request merged**